frontend/src/scenes/billing/Billing.tsx

```diff
@@ -241,14 +245,14 @@ export function Billing(): JSX.Element {
 
             <div className="flex justify-between">
                 <h2>Products</h2>
-                {isOnboarding && (
+                {isOnboarding && getUpgradeAllProductsLink() && (
                     <LemonButton
                         type="primary"
                         icon={<IconPlus />}
                         to={getUpgradeAllProductsLink()}
```

Comment:
Better to only call `getUpgradeAllProductsLink()` once

Reply:
fiiine I won't be lazy